src/coverlet.msbuild.tasks/MSBuildLogger.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using ILogger = Coverlet.Core.Logging.ILogger;
+
+namespace Coverlet.MSbuild.Tasks
+{
+    class MSBuildLogger : ILogger
+    {
+        private readonly TaskLoggingHelper _log;
+
+        public MSBuildLogger(TaskLoggingHelper log) => _log = log;
+
+        public void LogVerbose(string message) => _log.LogMessageFromText(message, MessageImportance.Low);
+
+        public void LogInformation(string message)=> _log.LogMessage(message);
+
+        public void LogWarning(string message) => _log.LogWarning(message);
+
+        public void LogError(string message) => _log.LogError(message);
+      
+        public void LogError(string message, Exception exception) => _log.LogErrorFromException(exception);
```

Comment:
better remove message here, I think it's better have two distinct method one with message and one with Exception what do you think?Here we cannot use message so or we call 2 method on task logger or remove first param

Reply:
I usually prefer to provide a message with the exception to help provide a little more info in case the exception is cryptic.   I thought I'd leave it here if other parts wanted to use it down the road, but I can remove it if you wish.

Reply:
I understand your idea, but maybe could be misleading, because if we don't know which logger we're using it's not clear why we pass a value that we'll not print on logger, I would remove

Reply:
Ah you could try a "fake" exception to better understand how msbuild logger print this message, but this is an exception and I think that message+stack will be enough